src/components/AllMonstersList.js

```diff
@@ -1,41 +1,52 @@
 import React, { useEffect, useState } from 'react'
 import styled from 'styled-components'
 import saveLocally from './lib/saveLocally'
+import OneMonsterItem from './OneMonsterItem'
 
 export default function AllMonstersList() {
-  const [monsterList, setMonsterList] = useState([])
+  const [monsterDetails, setMonsterDetails] = useState([])
 
   useEffect(() => {
-    const loadedArray = JSON.parse(localStorage.getItem('monsternames'))
+    const locallyLoadedMonsterNames = JSON.parse(
+      localStorage.getItem('monsternames')
+    )
 
-    if (!loadedArray || loadedArray.length === 0) {
+    if (!locallyLoadedMonsterNames || locallyLoadedMonsterNames.length === 0) {
       fetch(`https://www.dnd5eapi.co/api/monsters`)
```

Comment:
no reason to use template literals here

Reply:
Those are backticks ``. The fetch does require them. I just tried ;)